ert3/exceptions/_exceptions.py

```diff
@@ -0,0 +1,8 @@
+class Error(Exception):
```

Comment:
This name is very generic, should we call it `ERTError` or something similar instead?